spec/contracts.dd

```diff
@@ -99,18 +97,22 @@ body
     return cast(long)std.math.sqrt(cast(real)x);
```

Comment:
Btw shouldn't this page not also be updated in regards of [DIP1003](http://forum.dlang.org/post/esjclvbebspqkekjmwki@forum.dlang.org)?